pkg/monitoring/metrics/forklift-controller/metrics.go

```diff
@@ -0,0 +1,117 @@
+package forklift_controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	Succeeded = "Succeeded"
+	Failed    = "Failed"
+	Executing = "Executing"
+	Running   = "Running"
+	Pending   = "Pending"
+	Canceled  = "Canceled"
+	Blocked   = "Blocked"
+	Ready     = "Ready"
+	Deleted   = "Deleted"
+	Warm      = "Warm"
+	Cold      = "Cold"
+	Local     = "Local"
+	Remote    = "Remote"
+)
+
+var (
+	// 'status' - [ succeeded, failed, Executing, Canceled]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	migratioStatusCounter = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_migrations_status_total",
+		Help: "VM Migrations sorted by status and provider type",
+	},
+		[]string{
+			"status",
+			"provider",
+			"mode",
+			"target",
+		},
+	)
+
+	// 'status' - [ succeeded, failed, Executing, Running, Pending, Canceled, Blocked, Deleted]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	planStatusCounter = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_plans_status_total",
+		Help: "VM migration Plans sorted by status and provider type",
+	},
+		[]string{
+			"status",
+			"provider",
+			"mode",
+			"target",
+		},
+	)
+
+	// 'status' - [ succeeded, failed, Executing, Canceled]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	// 'plan' - [Id]
+	migrationDurationGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_migration_duration_in_seconds",
+		Help: "Duration of VM migrations in seconds",
+	},
+		[]string{"provider", "mode", "target", "plan"},
+	)
+
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	// 'plan' - [Id]
+	dataTransferredGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_data_transferred_in_megabytes",
+		Help: "Total data transferred during VM migrations in bytes",
+	},
+		[]string{
+			"provider",
+			"mode",
+			"target",
+			"plan",
+		},
+	)
+
+	// 'status' - [ succeeded, failed, Executing, Canceled]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	// 'plan' - [Id]
+	migratioPlanCorolationStatusnCounter = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_migrations_status_plan_correlation_total",
+		Help: "VM Migrations sorted by status and provider type correlated to the plan",
```

Comment:
I suggest:
VM Migrations sorted by status and provider type correlated to the plan ->
VM Migrations by status, provider type and plan.